src/background/adblocker.js

```diff
@@ -370,7 +368,7 @@ if (__PLATFORM__ === 'firefox') {
   function isExtensionRequest(details) {
     return (
       (details.tabId === -1 && details.url.startsWith('moz-extension://')) ||
-      details.originUrl.startsWith('moz-extension://')
+      details.originUrl?.startsWith('moz-extension://')
```

Comment:
This was a bug in the implementation - the `main_frame` requests do not have `originUrl`